go/pserver/client/c/test/test_train.py

```diff
@@ -31,6 +31,13 @@ def main():
 
     # create optimizer of new remote updater to pserver
     optimizer = paddle.optimizer.Momentum(momentum=0, learning_rate=1e-3)
+    optimizer.set_remote_optimizer_config(
```

Comment:
Well, converting V1 config to V2 may lose some of the configuration fields, but it's still possible. We always intend to keep the user interface simple, will add the conversions.

But I think we still should keep `set_remote_optimizer_config` or rename to `set_optimizer_config` for detailed optimizer configs.